ssl/build.info

```diff
@@ -33,7 +33,7 @@ SOURCE[../libssl]=\
         bio_ssl.c ssl_err.c tls_srp.c t1_trce.c ssl_utst.c \
         record/ssl3_buffer.c record/ssl3_record.c record/dtls1_bitmap.c \
         statem/statem.c record/ssl3_record_tls13.c record/tls_pad.c \
-        $KTLSSRC
+        tls_depr.c $KTLSSRC
```

Comment:
Hmmm, the trend so far has been to have `legacy` as part of the file name...  it might be a good idea to follow the same trend, so we can find them easily later with a `find . -name '*legacy*'`

Reply:
To me legacy does not necessarily mean deprecated, although obviously the two are related. OTOH, the "*_depr.c" suffix also has some precendent:

crypto/dh/dh_depr.c
crypto/dsa/dsa_depr.c
crypto/bn/bn_depr.c
crypto/rsa/rsa_deprc.